### ISSUE TITLE
Action link icon is not ignored by JAWS

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/vets-design-system-documentation#2208]
# Bug Report

- [X] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

From the 508 audit report for form 21-10210:

> The purpose of links cannot be determined from link text alone or from context.
> [2.4.4 Link Purpose (In Context) ](https://www.w3.org/WAI/WCAG21/quickref/#link-purpose-in-context)
> High
> Make sure that the purpose of each link can be determined from the link text alone or from context.
> On the submission screen, directly below the print this page button when navigating with the arrow keys “Link” is read using Jaws. This does not describe the purpose of the link

In the 508 audit report for form 21-10210, the 508 team identified at least two places where pseudo elements are being used for icons, and JAWS does not ignore them. However, since they are loading as pseudo elements (`::before`) there is no accessible text you can add to the icon and no `aria-hidden="true"` you can add to just the content of the pseudo element. 

I'm filing the issue here for the [action link](https://design.va.gov/storybook/?path=/docs/components-action-link--primary) (The other issue identified will be fixed once my team uses the right component.)

However, if there are other icons being loaded through pseudo elements, it would be good to look into updating those.

Here is a recording of what happens in our form:

https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/117098918/6776bfdf-4a38-4d40-9b7d-127efb02c2b5

When navving through a page going to every element using the down arrow, the 'focus' shifts from the 'print this page' button to the icon. You will hear the screen reader announce "Link" (because it is in a link) but it doesn't get the text of the link because it is a pseudo element. The next item is the link text itself. In the video I go back and forth, and you'll see that when you go back to the icon, from the link text, nothing is announced (This is what happens in other cases where the pseduo element is used next to a heading for example (or isn't a link).

I think the guidance here in the design system is a good one to follow when using icons. Use the `<i>` tags, and you can still put `aria-hidden="true"` on this tag, and JAWS will completely ignore it.

## What I expected to happen

I expected to be able to nav through a page using JAWS and have no empty announcements, empty focus, or non-descriptive announcements read aloud when passing icons

## Reproducing
- Formation version:
- Device: (e.g. iPhone 8, Macbook)
- Browser: (e.g. Firefox, IE 11)

Steps to reproduce:

1. You can see the video above in action on the confirmation page on form 21-10210 
2. Or you can go to [storybook's action link component](https://design.va.gov/components/link/action) and open up JAWS
3. With JAWS turned on, press the down key until you get to the Action link component
4. You'll here "Link" announced, and if you press the down key again, you'll here "Call to Action" announced

## Urgency
How urgent is this request? Please select the approriate option below and/or provide details
- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [X] This bug is not blocking any work - because it's already on production
- [ ] Other

### Details

We have sent this element through a 508 review before, but this is the first time they've commented on this component. This is also only happening JAWS, but I confirmed it to be happening in Edge, and Chrome. This issue does not occur in Firefox as it does treat the entire `<a>` as one component and does not focus on the icon itself.

**Comment by caw310** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/2208?planning-poker) with Zenhub @Andrew565 @ataker @harshil1793 @it-harrison @jamigibbs @micahchiang @nickjg231 @powellkerry @rmessina1010 @rsmithadhoc

**Comment by rsmithadhoc** [comment]
PR in review: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/994

**Comment by rsmithadhoc** [comment]
Hi @jeana-adhoc,

I put in a [fix](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/994) for this. There is a way to add alt text to it in the CSS. This will fix the issue in Edge/Chrome, however, it is not supported in some browsers, notably Firefox and Safari. So JAWS with Firefox would still see the icon.

Because this is not a traditional "component" (despite it being listed as one in the docs) and more of a style class that gets applied to a standard link, this is currently the best we can do. To have access to `<i>` and `aria-hidden`, it would need to be created as a new component.

You won't see the change live yet. This is a part of the CSS library, Formation, which won't release until December 4th due to some blocking work.

Thanks,
Ryan
